### PR TITLE
Clipboard target revisit

### PR DESCRIPTION
**Pull request by alvarmaciel**
Change the options for clipboard upload. add -f flag to add name

### CODE REVIEW

**Comment by mgaitan** [comment]
I like it 🌈 . It's simple and clean. 

As the semantic for file and directories are separated now, 
- I would rename `-o, --target` by `-d, --target-dir` , do you agree?
- we could remove the confirmation when "looks like a file"  
https://github.com/Shiphero/shbin/blob/a1a012a00f8e99263aec7a6a525956c8abace59a/shbin.py#L100-L105

also, with this change it would be the same  `shbin -x -f foo/bar.py`  than `shbin -x -f bar.py  -d foo`, right? If so, could we add a test for it and mention in the readme?

**Comment by alvarmaciel** [comment]
Thanks!!
> I would rename -o, --target by -d, --target-dir , do you agree?
>we could remove the confirmation when "looks like a file"
> with this change it would be the same shbin -x -f foo/bar.py than shbin -x -f bar.py -d foo, right? If so, could we add a test for it and mention in the readme?

Totally agree :notebook: for the next commit

**Comment by alvarmaciel** [comment]
> also, with this change it would be the same `shbin -x -f foo/bar.py` than `shbin -x -f bar.py -d foo`, right? If so, could we add a test for it and mention in the readme?

No, it's not the same.  `shbin -x -f foo/bar.py` will save to your user dir only with the flag -d you will set s path destination

File: shbin.py [hunk 1]
```diff
@@ -97,27 +98,25 @@ def main(argv=None) -> None:
...
+        files = [FakePath(file_name, content=content)]
 
     else:
         files = list(expand_paths(args["<path>"]))
-        dir_target = args["--target"] or ""
-
-        if (
-            files
-            and pathlib.PurePath(dir_target).suffix
-            and not Confirm.ask("--output looks like a file, not a directory. Are you sure?")
```
Comment (mgaitan) [inline shbin.py:117]: also remove the import

**Pull request merged**